test/Kestrel.Transport.BindTests/HandleInheritanceTest.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This shouldn't go in BindTests. In order to avoid conflicts, I don't want any tests running in parallel to the AddressRegistrationTests even if they bind to port 0.